test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1667,10 +1672,35 @@ def test_multiple_job_preemption_order(self):
 
         # Only One equivalence class for jid2 and jid3 is present since both
         # suspended jobs are of same type and running on same vnode
-        t = int(time.time())
+
         self.scheduler.log_match("Number of job equivalence classes: 2",
                                  max_attempts=10, starttime=self.t)
 
+        # Add a job to Queue state
+        a = {'Resource_List.ncpus': 1}
+        J = Job(TEST_USER, attrs=a)
+        jid6 = self.server.submit(J)
+        self.server.expect(JOB, {'job_state': 'Q'}, id=jid6)
+
+        # Set scheduling = false
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'scheduling': 'False'})
+
+        # Delete one running, one suspended job and one of high priority job
+        # This will leave 2 free nodes to pick up the suspended and queued job
+
+        self.server.deljob(jid1)
+        self.server.deljob(jid2)
+        self.server.deljob(jid5)
+
+        # Set scheduling = false
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'scheduling': 'True'})
+
+        # check jid3 and jid6 are running
```

Comment:
Either explain why you're checking if jid3/6 are running or remove this comment.  The two expect() lines say the same thing.